tutorials/plot_ecog.py

```diff
@@ -45,3 +48,15 @@
 subjects_dir = mne.datasets.sample.data_path() + '/subjects'
 fig = plot_trans(info, trans=None, subject='sample', subjects_dir=subjects_dir)
 mlab.view(200, 70)
+
+# If we want to make scatterplots with the figure, we can take a snapshot.
```

Comment:
yeah...the challenge here is that I don't really have the technical chops to debug vtk / mayavi bugs, so I can't be of too much help in making the fix :(